bin/deploy-storybook.sh

```diff
@@ -13,10 +13,18 @@ if [[ $IS_PONTOON ]]; then
   exit 0;
 fi
 
+# Skip builds for non-Mozilla users
+if [ "$CIRCLE_PROJECT_USERNAME" != "mozilla" ]; then
+  echo "Skipping Storybook deploy for non-Mozilla CircleCI";
+  exit 0;
+fi
+
 # HACK: Build static storybook with URL paths edited to include git hash
 HASH=$(git --no-pager log --format=format:"%H" -1)
 cp -r .storybook .storybook-$HASH
 sed -i "s#href=\"/static#href=\"/$HASH/static#g" .storybook-$HASH/preview-head.html
+# HACK: correct paths to experiment images in existing build
+sed -i "s#url(/static/#url(../#g" ./frontend/build/static/styles/experiments.css
```

Comment:
Tried it, but it broke gulp-rev-all - [so we reverted it](https://github.com/mozilla/testpilot/commit/17470d6520d197fb14c3d35b510ebb94a37a9a3d). Figured it was easier to hack it here than try to change that again